assets/src/js/admin/button.js

```diff
@@ -21,7 +21,8 @@ class GiveButton {
 
 	setBusyState() {
 		const busyIcon      = '<span class="dashicons dashicons-marker"></span>';
-		this.root.innerHTML = `${busyIcon} ${this.buttonText}`;
+		const text = this.root.querySelector('.give-button-text') ? `<span class="${this.buttonText}"></span>` : this.buttonText;
```

Comment:
instead of this why we can not use `this.buttonText` in the constructor.
```
this.buttonText    = this.root.textContent.trim();

// after update
this.buttonText    = this.root.querySelector('.give-button-text') ? `<span class="${this.buttonText}"></span>` : this.root.textContent.trim();
```

by the above way, we do not have to change code at multiple places.

Reply:
This was something I explored, but realized it adds more complexity to the updateText() function, where as this implementation does not impact the updateText() function at all. I just committed one possible solution, refactor both querySelectors into the constructor as you suggested but under a new property called "text" which refers to either the buttonText string of the full text span, depending on what is present.